cmd/instance/instance.go

```diff
@@ -54,6 +54,7 @@ func init() {
 	instanceCreateCmd.Flags().StringVarP(&firewall, "firewall", "l", "", "the instance's firewall you can use the Name or the ID")
 	instanceCreateCmd.Flags().StringVarP(&tags, "tags", "g", "", "the instance's tags")
 	instanceCreateCmd.Flags().StringVarP(&privateIPv4, "private_ipv4", "", "", "Private IPv4 address")
+	instanceCreateCmd.Flags().StringVarP(&reservedIPv4, "reservedip", "", "", "Reserved IPv4 address")
```

Comment:
"reserved_ipv4" is the correct param to be passed to the api-go.
do we wanna map it as `--reserved-ipv4`?